core/services/relay/evm/functions/offchain_config_digester.go

```diff
@@ -0,0 +1,134 @@
+package functions
+
+import (
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/smartcontractkit/libocr/gethwrappers2/exposedocr2aggregator"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/types"
+)
+
+var (
+	_                     types.OffchainConfigDigester = FunctionsOffchainConfigDigester{}
+	configDigestArgs                                   = makeConfigDigestArgs()
+	FunctionsDigestPrefix                              = types.ConfigDigestPrefixEVM
+	ThresholdDigestPrefix                              = types.ConfigDigestPrefix(7)
```

Comment:
Can you add a comment explaining why this is done? So future readers are not confused too much.

Reply:
These magic numbers 7, 8 are unclear to me.

Reply:
They were the next prefixes in the sequence that are currently un-used by libocr.  [See this PR](https://github.com/smartcontractkit/libocr/pull/69) (the linked PR probably won't ever get merged, but explains the reasoning)